src/ui/components/navigation/navigationcomponent.js

```diff
@@ -338,7 +339,7 @@ export default class NavigationComponent extends Component {
     for (let i = 0; i < this._tabOrder.length; i++) {
       let tab = this._tabs[this._tabOrder[i]];
       if (tab !== undefined) {
-        tab.url = this.generateTabUrl(tab.baseUrl, this.getUrlParams());
+        tab.url = addParamsToUrl(tab.baseUrl, { tabOrder: this._tabOrder });
```

Comment:
sorry missed this the first time but this also needs referrerPageUrl as well as apiContext (just the additional 1.5.2 stuff from the old generateTabUrl)